From d4c34c201c8edd0db0e77252b227bff7a109f126 Mon Sep 17 00:00:00 2001
From: Igal Chernobelsky <igalc@ti.com>
Date: Sun, 17 Feb 2013 14:27:37 +0200
Subject: [PATCH] hostapd.conf: update max_num_sta and model_number

Set max_num_sta to 5 as FW supports connection of 5 STA-s
and model_number to wl18xx.

Change-Id: I590b341635f80db6d30c35ddd7250610fd817b27
Signed-off-by: Igal Chernobelsky <igalc@ti.com>
---
 mac80211/config/hostapd.conf |    5 ++---
 1 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/mac80211/config/hostapd.conf b/mac80211/config/hostapd.conf
index 26fed92..f59a917 100644
--- a/mac80211/config/hostapd.conf
+++ b/mac80211/config/hostapd.conf
@@ -40,7 +40,7 @@ interface=wlan1
 ssid=AndroidAP
 ieee80211d=1
 country_code=US
-max_num_sta=5
+max_num_sta=8
 macaddr_acl=0
 auth_algs=3
 tx_queue_data3_aifs=7
@@ -100,8 +100,7 @@ ht_capab=[SHORT-GI-20][GF]
 device_name=Blaze
 manufacturer=TexasInstruments
 model_name=TI_Connectivity_module
-model_number=wl12xx
+model_number=wl18xx
 serial_number=12345
 device_type=0-00000000-0
 config_methods=virtual_display virtual_push_button keypad
- 
-- 
1.7.9

